layout-server/src/main/java/org/finos/vuu/layoutserver/controller/LayoutController.java

```diff
@@ -98,6 +98,8 @@ public void updateLayout(@PathVariable UUID id, @RequestBody @Valid LayoutReques
     @ResponseStatus(HttpStatus.NO_CONTENT)
     @DeleteMapping("/{id}")
     public void deleteLayout(@PathVariable UUID id) {
+        // Generate a 404 if layout doesn't exist
+        layoutService.getLayout(id);
```

Comment:
Do we want an exception to be thrown or do we want it to just silently do nothing?

Reply:
I'm inclined to say just logging it would be enough. DOn't see much value in informing the user - something has obviously gone wrong if this happens, so we should capture it, but its not going to affect the users experience.